### ISSUE TITLE
The hackage version is broken

### ISSUE THREAD

**Issue by bartavelle** [jano017/Discord.hs#15]
And so is the current master:

https://hackage.haskell.org/package/discord-hs-0.3.2/src/src/Network/Discord/Rest/Prelude.hs

`Fetched` has kind `*`, and, AFAIK, can't be a Functor or a Comonad.

**Comment by EpicOrange** [comment]
Yes, #13 should probably be reverted. `Fetched` isn't being used as a Functor or Comonad anywhere at the moment.
